pkg/shared/accountmgmtutil/ams.go

```diff
@@ -68,72 +58,57 @@ func fetchOrgQuotaCost(ctx context.Context, conn connection.Connection) (*amscli
 
 }
 
-func GetUserSupportedInstanceTypes(ctx context.Context, spec *remote.AmsConfig, conn connection.Connection) (quota []QuotaSpec, err error) {
+func FetchQuotaCost(f *factory.Factory, billingModel string, cloudAccountID string, marketplace string, spec *remote.AmsConfig) (userQuotaSpec *QuotaSpec, err error) {
```

Comment:
@wtrocki . Just wanted to get this logic verified.
Also, what should be the case if user specifying `--billing-model` as marketplace should make cloudaccountID and marketplace compulsory?